src/binaryen-c.h

```diff
@@ -3175,6 +3175,9 @@ BINARYEN_API char* BinaryenModuleAllocateAndWriteText(BinaryenModuleRef module);
 BINARYEN_API char*
 BinaryenModuleAllocateAndWriteStackIR(BinaryenModuleRef module, bool optimize);
 
+// Deserialize a module from binary form.
+BINARYEN_API BinaryenModuleRef BinaryenModuleReadWithFeatures(
+  char* input, size_t inputSize, BinaryenFeatures featureSet);
```

Comment:
Actually maybe this could go second, after the first form, and the comment could be non-identical and explain the difference?

Reply:
(Maybe something like `Deserialize a module from binary form while enabling a particular set of features (this differs from `BinaryenModuleRead` which assumes the MVP featureset).

Reply:
Done